Library/Homebrew/test/caveats_spec.rb

```diff
@@ -26,4 +26,53 @@ def caveats
       expect(described_class.new(f)).not_to be_empty
     end
   end
+
+  describe "#caveats" do
+    context "when f.plist is not nil" do
+      it "prints plist startup information when f.plist_startup is not nil" do
+        p = formula do
```

Comment:
If I use `f = formula do`, https://github.com/mansimarkaur/brew/blob/4b4762f5cba3d183d5347497df1bf8a524ce035e/Library/Homebrew/test/caveats_spec.rb#L6 interferes with this definition.